src/app/public/modules/datepicker/datepicker.component.scss

```diff
@@ -1,4 +1,5 @@
 @import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/variables";
```

Comment:
Was this imported by accident? I don't see any changes to the scss file other than this line.